packages/e2e-tests/src/specs/dashboard/dashboard.js

```diff
@@ -28,6 +28,8 @@ import {
 
 const percyCSS = `.dashboard-grid-item-date { display: none; }`;
 
+jest.retryTimes(3, { logErrorsBeforeRetry: true });
```

Comment:
Curious if you've seen any reties show up in the logs or artifact files.

Reply:
Yes, they do show up there. Even on this PR here you can see a bunch of retries in the logs and the artifact zip